### PR TITLE
DRA: log output

### PR DESCRIPTION
**Pull request by pohly**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Some test log output wasn't as useful as it should have been. See commit messages for a before/after comparison.

#### Special notes for your reviewer:

Found while working on https://github.com/kubernetes/kubernetes/pull/125488

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

/sig node

/assign @byako 

@byako: can you perhaps review? One change is about code that you wrote.

### CODE REVIEW

File: test/utils/ktesting/tcontext.go [hunk 1]
```diff
@@ -241,7 +242,19 @@ func Init(tb TB, opts ...InitOption) TContext {
...
+				// adds a <"type"> prefix, which is too long
+				// for simple values.
+				switch v := v.(type) {
+				case int, int32, int64, uint, uint32, uint64, float32, float64, bool:
+					return fmt.Sprintf("%v", v)
+				case string:
+					return v
+				default:
+					return strings.TrimSpace(format.Object(v, 1))
+				}
```
Comment (bart0sh) [inline test/utils/ktesting/tcontext.go:257]: This seems to influence a lot of existing code. Would it make sense to do this in a separate PR?

Comment (pohly) [reply test/utils/ktesting/tcontext.go:257]: > This seems to influence a lot of existing code. 

Only tests, and only those using ktesting.

> Would it make sense to do this in a separate PR?

To simplify reverting? That can be done on a per-commit basis if needed, too.

I think a separate PR is overkill (each PR comes with its own costs), but I can create one if you prefer that.

Comment (pohly) [reply test/utils/ktesting/tcontext.go:257]: ... in particular when we have many unrelated flakes which need retesting :cry: 

/retest

Comment (pohly) [reply test/utils/ktesting/tcontext.go:257]: @bart0sh: what do you think?

Can you review? It doesn't have to be @byako .

Comment (bart0sh) [reply test/utils/ktesting/tcontext.go:257]: sure,
/lgtm
